x-pack/legacy/plugins/reporting/export_types/png/server/execute_job/index.ts

```diff
@@ -54,13 +54,14 @@ export const executeJobFactory: QueuedPngExecutorFactory = async function execut
           job.layout
         );
       }),
-      map(({ buffer, warnings }) => {
+      map(({ base64, warnings }) => {
```

Comment:
I had to rename this - it was very unclear that the return payload actually is a `string`